var/spack/repos/builtin/packages/raja/package.py

```diff
@@ -246,19 +308,50 @@ def initconfig_package_entries(self):
         else:
             entries.append(cmake_cache_option("ENABLE_EXERCISES", "+exercises" in spec))
 
+        # TODO: Treat the workaround when building tests with spack wrapper
+        #       For now, removing it to test CI, which builds tests outside of wrapper.
```

Comment:
Is this still an issue?

Reply:
I’ll have to look into the origin of this. Also, I never build the tests with spack wrapper. This may simply need to a clean-up.

Reply:
The part below seems stale, since the wrappers remove -Werror by default.  If the TODO can go too that would be great, either way want to know what's going on.